### PR TITLE
gh-release-make-flags-optional

### PR DESCRIPTION
**Pull request by f1sh1918**
make flags optional

### CODE REVIEW

**Review by ztefanie — CHANGES_REQUESTED** [review]
I get an error, when trying to run this `v0 release create all 1 1 --deliverino-private-key my-key --should-use-predefined-release-notes`

error: option '--should-use-predefined-release-notes <should-use-predefined-release-notes>' argument missing

**Comment by f1sh1918** [comment]
> I get an error, when trying to run this `v0 release create all 1 1 --deliverino-private-key my-key --should-use-predefined-release-notes`
> 
> error: option '--should-use-predefined-release-notes ' argument missing

how can you run this without `--owner --repo` and @ztefanie ?

**Comment by f1sh1918** [comment]
i pushed new commit @ztefanie thx for the hint

File: src/release/program-github-release.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ export default (parent: Command) =>
     )
     .requiredOption('--owner <owner>', 'owner of the current repository, usually "Integreat"')
     .requiredOption('--repo <repo>', 'the current repository, should be integreat-app')
-    .requiredOption('--production-release <production-release>', 'whether this is a production release or not')
-    .requiredOption('--should-use-predefined-release-notes <should-use-predefined-release-notes>', 'whether predefined release notes should be used or not. Release notes have to be passed if true.')
+    .option('--production-release', 'whether this is a production release or not. If unset false')
+    .option('--should-use-predefined-release-notes', 'whether predefined release notes should be used or not. Release notes have to be passed if true.')
```
Comment (ztefanie) [inline src/release/program-github-release.ts:14]:
```suggestion
    .option('--should-use-predefined-release-notes', 'whether predefined release notes should be used or not. Release notes have to be passed if set. If unset false')
```

**Review by ztefanie — APPROVED** [review]
Just one little comment.

**Pull request merged**